.monorepolint.config.mjs

```diff
@@ -88,21 +88,29 @@ export default {
     packageEntry({
       options: {
         entries: {
+          type: "module",
+          main: "dist/cjs/index.cjs",
+          module: "dist/esm/index.mjs",
+          types: "dist/esm/index.d.ts",
+          sideEffects: false,
+          publishConfig: {
+            access: "public",
+          },
           // @turf/turf is commonly consumed through CDNs, moving this output file is a breaking change for anyone
           // who has a hardcoded reference to this specific file, instead of letting the CDN pick the path.
           // Example of a URL that will break: https://unpkg.com/@turf/turf/dist/turf.min.js
           // Example of a URL that will keep working: https://unpkg.com/@turf/turf
           browser: "turf.min.js",
-          files: ["dist", "index.d.ts", "turf.min.js"],
+          files: ["dist", "turf.min.js"],
           exports: {
             "./package.json": "./package.json",
             ".": {
               import: {
-                types: "./dist/esm/index.d.mts",
+                types: "./dist/esm/index.d.ts",
                 default: "./dist/esm/index.mjs",
               },
               require: {
-                types: "./dist/cjs/index.d.ts",
+                types: "./dist/cjs/index.d.cts",
```

Comment:
I was a little surprised to see `.cts` as an extension here, but if that's what tsup gives us that seems fine.

Reply:
This is easy enough to fix. Will change to use more traditional file extensions in both flavour dist/ subdirectories.

Reply:
Actually, can't do this just at the moment. tsup will override the .cjs extension to .js, but not .d.cts to .d.ts Depends on https://github.com/egoist/tsup/issues/939